app/assets/stylesheets/application.css.scss

```diff
@@ -36,6 +36,10 @@ h2 {
   color: #ffffff;
 }
 
+h3 {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.